### PR TITLE
[WNMGDS] Accordion HCM support

### PR DESCRIPTION
**Pull request by ericwbailey**
## Summary

[JIRA ticket](https://jira.cms.gov/browse/WNMGDS-1308).

This PR updates [the Accordion component](https://design.cms.gov/components/accordion/) to support Windows High Contrast Mode.

### Added

Media queries that update the Accordion's header button to better describe its size and interactable status.

### Before

Screenshot of the accordion component with Windows High Contrast Mode active. The accordion heading backgrounds are invisible.

### After

https://user-images.githubusercontent.com/634191/150204289-f32208dd-687f-47c1-ad87-7d0073eb9ca6.mp4

Accordion header button is visible in its resting state, and hover/focus interactions are now added.

### CODE REVIEW

File: packages/design-system/src/styles/components/_Accordion.scss [hunk 1]
```diff
@@ -41,13 +41,32 @@ $accordion-header-font-family: $font-sans;
...
+    background-color: LinkText;
+    color: window;
+
+    &:hover,
+    &:focus {
+      -ms-high-contrast-adjust: none;
+      background-color: window;
+      color: LinkText;
+      outline-offset: -4px;
+      outline: 4px solid LinkText;
```
Comment (line47) [inline packages/design-system/src/styles/components/_Accordion.scss:65]: Should we consider using our spacer variable here for the 4px?

Comment (ericwbailey) [reply packages/design-system/src/styles/components/_Accordion.scss]: Oh, good call. Added!

**Review by scheul93 — APPROVED** [review]
A great improvement!

**Review by forrestbaer — APPROVED** [review]
looks great

**Pull request merged**